bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/dto/clip2/enums/RecallAction.java

```diff
@@ -22,9 +22,15 @@
  */
 @NonNullByDefault
 public enum RecallAction {
+    // values for normal scenes
     ACTIVE,
     DYNAMIC_PALETTE,
-    STATIC;
+    STATIC,
+    // values for smart scenes
+    ACTIVATE,
```

Comment:
It seems slightly confusing to have `ACTIVE` and `ACTIVATE` in the same enum. Can you add some context about their meaning?

Reply:
See Scene ‘recall’ DTO field here..
https://developers.meethue.com/develop/hue-api-v2/api-reference/#resource_scene__id__put

See Smart Scene ‘recall’ DTO field here..
https://developers.meethue.com/develop/hue-api-v2/api-reference/#resource_smart_scene__id__put

Reply:
Thanks. So they are actually from two different DTO's. Same attribute name, but different definitions:

**recall:** _(object)_
- **action:** _(one of active, dynamic_palette, static)_
When writing active, the actions in the scene are executed on the target. dynamic_palette starts dynamic scene with colors in the Palette object.

**recall:** _(object)_
- **action:** _(one of activate, deactivate)_
Activate will start the smart (24h) scene; deactivate will stop it

Perhaps it would be better to separate them accordingly? There is no guarantee that there will not be introduced new values in the future which could overlap.

Reply:
A scene is something that the user activates directly, whereas a smart scene is something which the user may "activate", but which is only really activated when some other event occurs e.g. wake-up alarm.

> Perhaps it would be better to separate them accordingly?

Do you mean to have separate enums? (In which case ok). Or to have a fully separate channel. (In which case we need to discuss as I was previously criticised for wanting too many channels).

Reply:
Yes, I would only suggest to use separate enums according to the DTO's, but stick with the current modelling in openHAB, i.e. keep only the existing channel.

Reply:
@jlaur thanks to your comment I realised there is a problem still to be solved — namely 
- for normal scenes one can only activate it (there is no such thing as deactivating a scene)
- by contrast for smart scenes it requires both the possibility to activate and deactivate it
- unlike normal scenes where only one can be active at any time, I can imagine that there may be more than one smart scene “active” at the same time; each one waiting to be triggered by its own smart criteria.

So I think I need to modify the architecture for this PR so that instead of having multiple dynamic state options one for each scene on a single channel, it should instead have multiple dynamic channels one for each smart scene.